dashboard/test/ui/features/platform/policy_compliance.feature

```diff
@@ -145,3 +145,39 @@ Feature: Policy Compliance and Parental Permission
   Scenario: Existing under 13 account in Colorado should not be locked out.
     Given I create a young student in Colorado who has never signed in named "Sally Student" before CPA exception and go home
     Given I am on "http://studio.code.org/home"
+
+  Scenario: Student should not be able to connect a third-party account until their account is unlocked
+    Given I create an old account for a young student in Colorado who has never signed in named "Coco Student" and go home
+
+    # Find the locked buttons to connect an account
+    Given I am on "http://studio.code.org/users/edit?cpa-partial-lockout=1&cpa_schedule={%22cpa_new_user_lockout%22:%222023-07-05T23:15:00%2B00:00%22,%22cpa_all_user_lockout_warning%22:%222024-07-01T00:00:00%2B00:00%22,%22cpa_all_user_lockout%22:%222024-07-01T00:00:00%2B00:00%22}"
+    Then I wait to see "#manage-linked-accounts"
+    Then I wait until "form[action=\'/users/auth/google_oauth2?action=connect\'] button" is disabled
+
+    # Navigate the lockout process
+    Given I am on "http://studio.code.org/lockout"
```

Comment:
I don't think a user can navigate to this page unless they are locked out? This test is meant to simulate a user who is in the warning/pre-lockout phase, right?

Why navigate to the lockout page? Doesn't `users/edit` have the widget for requesting permission?

Reply:
Absolutely right. Updating it to work with the account settings page instead.